src/main/java/hudson/plugins/claim/ClaimPublisher.java

```diff
@@ -63,11 +74,10 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
             }
         }
 
-        return true;
     }
 
     public BuildStepMonitor getRequiredMonitorService() {
-        return BuildStepMonitor.STEP;
+        return BuildStepMonitor.NONE;
     }
```

Comment:
Uhm, I would discuss this change. It may cause some issues, because actually the original behavior synchronizes builds within the project. So for free-style jobs the behavior changes:
* Original behavior: build #N always runs the publisher after build #N-1 
* New behavior: build #N and build #N-1 can be completed independently

@ki82 Is there an impact on plugin use-cases?